capstone-frontend/src/components/AdminClubPage.jsx

```diff
@@ -20,23 +20,24 @@ class AdminClubPage extends Component {
   fetchData = () => {
     let requesters = [];
     let members = [];
-    fetch(`/api/clubs?id=${this.props.id}`)
+    let club; 
+    fetch(`/api/clubs?id=${this.props.match.params.id}`)
       .then(response => response.json()).then(clubs => {
-        const club = clubs[0];
-        this.setState({ club, requesters: [], members: [] });
+        club = clubs[0];
```

Comment:
nit: Can this remain a `const`?

Reply:
Sure! Was changing things around to test, and forgot to revert this to a const.